src/olympia/addons/views.py

```diff
@@ -474,7 +474,7 @@ def contribute(request, addon):
                                              'paykey': ''}),
                                  content_type='application/json')
 
-    contribution_uuid = hashlib.md5(str(uuid.uuid4())).hexdigest()
+    contribution_uuid = hashlib.sha256(str(uuid.uuid4())).hexdigest()
```

Comment:
Just made sure... I was worried this had any implications on PayPal but it looks like they allow pretty much anything for this identifier, more importantly was our database side and it is a varchar(255) so we're safe.